main/admin/access_url_add_usergroup_to_url.php

```diff
@@ -87,7 +87,7 @@
                   <?php
                     echo Display::get_alphabet_options($firstLetterUserGroup);
                     echo Display::get_numeric_options(0, 9, $firstLetterUserGroup);
-                  ?>
+                    ?>
```

Comment:
!? That's weird... why would it recommend to end the php tag at this level of indentation?